pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1254,6 +1254,32 @@ static __always_inline bool is_compat(struct task_struct *task)
 #endif
 }
 
+static __always_inline int get_syscall_id_from_regs(struct pt_regs *regs)
```

Comment:
Let's use this function in sys_exit as well, since we have the same code there too

Reply:
Done.